eoscostarica/eosio-dashboard
# chore(button): improve Execute transaction button #870

###  improve Execute transaction button

### What does this PR do?

- Resolve #870 

### Steps to test

1. Go to https://mainnet.eosio.online/accounts and see contract tables `table` input form and execute transaction in` Contract Actions`

#### CheckList
